src/translations/pattern.translations..js

```diff
@@ -0,0 +1,4 @@
+export const patternTranslations = {
```

Comment:
We don't need `translations` folder, it will confuse developers a little bit because we don't translate anything it is better to rename it to the `users.messages` because it just reflects messages